docs/developer/devicesimulator.md

```diff
@@ -46,14 +281,56 @@ On each remote host run `devicesimulator` with at least two parameters:
     > bin/devicesimulator --count=200 --initial-device-index=200 \# device-00200 to device-00399
     > 
     > bin/devicesimulator --count=200 --initial-device-index=400 \# device-00400 to device-00599
- 
-Per each device a `podman event` will be created to consume its events. These tasks will never finish and will consume a considerable amount of memory. As a workaround run the following script on each host:
 
+When running remotely it is important to consider [connection limits](devicesimulator.md#system-level-connection-limits) when running large scale performance tests.
+
+**Source IP Considerations for Remote Hosts**: When distributing device simulation across multiple remote hosts, each host can benefit from using multiple source IP addresses to avoid ephemeral port exhaustion. This is especially important when running high device counts per host (>10,000 devices). Configure network aliases on each remote host before running the simulator:
```

Comment:
It is important to note that we need to verify we do not cause any IP address conflicts with other hosts in the network.
We might want to consider creating a virtual nic (for example bridge) and assigning IP addresses to it.  These IPs should be in a network that doesn't currently exist.